lib/jeckle/mapping.rb

```diff
@@ -0,0 +1,30 @@
+module Jeckle
+  module Mapping
+    class CustomAttributeMapping
```

Comment:
Do we need to call it `CustomAttributeMapping` inside a `Mapping` namespace? What if we remove `Mapping` namespace?